modules/appeals_api/app/services/appeals_api/evidence_submission_request_validator.rb

```diff
@@ -72,8 +73,8 @@ def invalid_review_option_error
     def invalid_veteran_id_error
       {
         title: 'unprocessable_entity',
-        detail: I18n.t('appeals_api.errors.mismatched_ssns'),
-        code: 'DecisionReviewMismatchedSSN',
+        detail: I18n.t('appeals_api.errors.mismatched_veteran_identifier'),
+        code: 'DecisionReviewMismatchedVeteranIdentifier',
```

Comment:
We don't want to do this yet, as this change affects all NOD versions, and we have a contract to keep for production even if we don't have any clients using it.

We should either make this respond with this new error only for V2, or wait until we get confirmation that we can remove NOD v1.

Reply:
Right. Ok. Things get murky in our handling of the ev sub validation, but I'll make it work.